Tests/WorkspaceTests/ManifestSourceGenerationTests.swift

```diff
@@ -239,24 +240,25 @@ class ManifestSourceGenerationTests: XCTestCase {
             let package = Package(
                 name: "MyPackage",
                 dependencies: [
-                   .package(url: "/foo1", from: "1.0.0"),
-                   .package(url: "/foo2", .revision("58e9de4e7b79e67c72a46e164158e3542e570ab6")),
-                   .package(path: "../foo3"),
-                   .package(path: "/path/to/foo4"),
-                   .package(url: "/foo5", .exact("1.2.3")),
-                   .package(url: "/foo6", "1.2.3"..<"2.0.0"),
-                   .package(url: "/foo7", .branch("master")),
-                   .package(url: "/foo8", .upToNextMinor(from: "1.3.4")),
-                   .package(url: "/foo9", .upToNextMajor(from: "1.3.4")),
-                   .package(path: "~/path/to/foo10"),
-                   .package(path: "~foo11"),
-                   .package(path: "~/path/to/~/foo12"),
-                   .package(path: "~"),
-                   .package(path: "file:///path/to/foo13"),
+                   .package(url: "https://example.com/MyPkg1", from: "1.0.0"),
+                   .package(url: "https://example.com/MyPkg2", .revision("58e9de4e7b79e67c72a46e164158e3542e570ab6")),
+                   .package(url: "https://example.com/MyPkg5", .exact("1.2.3")),
+                   .package(url: "https://example.com/MyPkg6", "1.2.3"..<"2.0.0"),
+                   .package(url: "https://example.com/MyPkg7", .branch("main")),
+                   .package(url: "https://example.com/MyPkg8", .upToNextMinor(from: "1.3.4")),
+                   .package(url: "ssh://git@example.com/MyPkg9", .branch("my branch with spaces")),
+                   .package(url: "../MyPkg10", from: "0.1.0"),
+                   .package(path: "../MyPkg11"),
+                   .package(path: "packages/path/to/MyPkg12"),
```

Comment:
feels like we lost some of the test cases here, should we bring them back?

Reply:
I removed the ones that didn't make sense, e.g. `~`.  When I added those I wasn't really thinking about what made sense to create, so I replaced them with ones that make more sense.  I can add them back if you think it makes sense.

Reply:
I can see the case for `~` with hand-authored packages, but for the mechanically generated manifests that doesn't make much sense.  But I can add back the test.

`file://` doesn't make sense at all for a `path:` so we should remove that.  I can file an issue to do that.  I'll add back the `~` tests.

Reply:
Actually I'll add them all back and then file an issue to remove `file://`.

Reply:
Added them back in 0dc348858a5100cb018de45d1d72e6a45eb8a0f0